File: NutriTechBackOffice.Domain/Entities/Paciente.cs
using System;
using System.Collections.Generic;
using System.Text;
using Google.Cloud.Firestore;

namespace NutriTechBackOffice.Domain.Entities
{
    [FirestoreData]
    public class Paciente : User
    {
        [FirestoreProperty]
        public float? Altura { get; set; }

        [FirestoreProperty]
        public float? Peso { get; set; }

        [FirestoreProperty]
        public float? MedidaCintura { get; set; }

        [FirestoreProperty]
        public string TipoAlimentacion { get; set; }

        [FirestoreProperty]
        public PlanAsignacion PlanAsignado { get; set; }

    }
}

File: NutriTechBackOffice/ClientApp/src/app/interfaces/paciente.ts
import { PlanAsignado } from "./plan-asignado";
import { LastUpdated, User } from "./user";

export interface Paciente extends User {
  altura: number;
  peso: number;
  medidaCintura: number;
  tipoAlimentacion: string;
  planAsignado: PlanAsignado;
  lastAssignment: LastUpdated;
}
